.travis.yml

```diff
@@ -17,6 +17,6 @@ install:
 script:
     - qmake
     - make
-    - "! (astyle *.h *.cpp -r --dry-run --options=astylerc | grep formatted)"
+    - "! (astyle "*.h" "*.cpp" -r --dry-run --options=astylerc | grep formatted)"
```

Comment:
...I think I was wrong about this, drop the quotation marks you added in and it should work now!